python/Ganga/Core/InternalServices/Coordinator.py

```diff
@@ -234,10 +180,10 @@ def enableInternalServices():
     Repository_runtime.bootstrap()
 
     # make sure all required credentials are valid
-    missing_cred = getMissingCredentials()
-    if missing_cred:
-        log.error("The following credentials are still required: %s."
-                  "Make sure you renew them before reactivating this session" % ','.join(missing_cred))
+    invalid_afs = [afsToken for afsToken in credential_store.get_all_matching_type(AfsToken) if not afsToken.is_valid()]
+
+    if invalid_afs:
+        log.error('No valid AFS token was found. Please re-authorise before reactivating this session.')
```

Comment:
@drmarkwslater Would it be sensible to suggest that Ganga attempt (via some mechanism) to renew it's token as I think 5 automated renewals are typically allowed which can extend a session to 5 days without a password being needed?
In this case are we going to mark the Registries as requiring an AFS token if they're on an AFS filesystem?